modules/xfem/test/tests/bimaterials/glued_interface_2d.i

```diff
@@ -0,0 +1,315 @@
+  ymax = 10
+  elem_type = QUAD4
+  displacements = 'disp_x disp_y'
+[]
+
+[MeshModifiers]
+  [./left_bottom]
+    type = AddExtraNodeset
+    new_boundary = 'left_bottom'
+    coord = '0.0 0.0'
+  [../]
+  [./left_top]
+    type = AddExtraNodeset
+    new_boundary = 'left_top'
+    coord = '0.0 10.'
+  [../]
+[]
+
+[AuxVariables]
+  [./ls]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[AuxKernels]
+  [./ls_function]
+    type = FunctionAux
+    variable = ls
+    function = ls_func
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+[]
+
+[Functions]
+  [./ls_func]
+    type = ParsedFunction
+    value = 'y-5.0+1.1*t'
+  [../]
+[]
+
+[AuxVariables]
+  [./stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./stress_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./stress_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./a_strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./a_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./a_strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./b_strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./b_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./b_strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+  [../]
+[]
+
+[AuxKernels]
+  [./stress_xx]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    index_i = 0
+    index_j = 0
+    variable = stress_xx
+  [../]
+  [./stress_yy]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    index_i = 1
+    index_j = 1
+    variable = stress_yy
+  [../]
+  [./stress_xy]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    index_i = 0
+    index_j = 1
+    variable = stress_xy
+  [../]
+  [./a_strain_xx]
+    type = RankTwoAux
+    rank_two_tensor = A_total_strain
+    index_i = 0
+    index_j = 0
+    variable = a_strain_xx
+  [../]
+  [./a_strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = A_total_strain
+    index_i = 1
+    index_j = 1
+    variable = a_strain_yy
+  [../]
+  [./a_strain_xy]
+    type = RankTwoAux
+    rank_two_tensor = A_total_strain
+    index_i = 0
+    index_j = 1
+    variable = a_strain_xy
+  [../]
+  [./b_strain_xx]
+    type = RankTwoAux
+    rank_two_tensor = B_total_strain
+    index_i = 0
+    index_j = 0
+    variable = b_strain_xx
+  [../]
+  [./b_strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = B_total_strain
+    index_i = 1
+    index_j = 1
+    variable = b_strain_yy
+  [../]
+  [./b_strain_xy]
+    type = RankTwoAux
+    rank_two_tensor = B_total_strain
+    index_i = 0
+    index_j = 1
+    variable = b_strain_xy
+  [../]
+[]
+
+[Constraints]
+  [./dispx_constraint]
+    type = XFEMSingleVariableConstraint
+    use_displaced_mesh = false
+    variable = disp_x
+    use_penalty = true
+    alpha = 1e5
+  [../]
+  [./dispy_constraint]
+    type = XFEMSingleVariableConstraint
```

Comment:
Do you need `geometric_cut_userobject = 'level_set_cut_uo'` in these Constraint blocks?

Reply:
this is the test that I forget to delete.